packages/portal/src/components/item/ItemSelectCheckbox.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <b-form-checkbox
+    v-model="selected"
+    class="item-select-checkbox position-absolute"
+    :class="{ active: selected }"
+    @change="toggleItemSelection"
+  >
+    <span
+      class="m-3 position-relative d-inline-block"
+      :class="selected ? 'icon-select-circle' : 'icon-select-circle-outlined'"
+    />
+    <span class="visually-hidden">{{ selectCheckboxLabel }}</span>
```

Comment:
Is repeating the item title the best choice of label for the checkbox in terms of a11y?

Reply:
I think the item title is a clear label for the user to determine they want to select/deselect it.
In the select state the only interactive element on the item card is the checkbox. When tabbing and using a screen reader, and tabbing over the cards (i.e. checkboxes) it will read out this label and the checked state ('ticked' or 'unticked').
I do see a possible refactor: to use the card title as checkbox label (similar to it being a link in the regular state). Then we don't need duplicate card title and checkbox label elements. I opened a separate PR to do so: https://github.com/europeana/portal.js/pull/2631